src/divider.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
+import ThemeManager from './styles/theme-manager';
+import styleUtils from './utils/styles';
+
+const propTypes = {
+  /**
+   * CSS class that will be add to the divider's root element
+   */
+  className: React.PropTypes.string,
+
+  /**
+   * If true, the divider will be indented by 72px
+   */
+  inset: React.PropTypes.bool,
+
+  /**
+   * Override the inline-styles of the list divider's root element.
+   */
+  style: React.PropTypes.object,
+};
+
+const contextTypes = {
+  muiTheme: React.PropTypes.object,
+};
+
+const childContextTypes = {
+  muiTheme: React.PropTypes.object,
+};
+
+const defaultProps = {
+  inset: false,
+};
+
+const Divider = ({inset, style, ...other}, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
```

Comment:
I can revert this latest commit and we can explore this stateless with context approach separately if you want. This context stuff is still a little new to me. material-ui is the first time I'm really running into it.